core/store/src/main/java/uk/gov/gchq/gaffer/store/serialiser/lengthvalue/EdgeIdSerialiser.java

```diff
@@ -89,4 +89,9 @@ public EdgeId deserialiseEmpty() throws SerialisationException {
     public boolean preservesObjectOrdering() {
         return false;
```

Comment:
Please could you also update this to:

return null != vertexSerialiser && vertexSerialiser.preservesObjectOrdering();